packages/p2p/src/pages/advertiser-page/advertiser-page.jsx

```diff
@@ -101,28 +101,43 @@ const AdvertiserPage = () => {
 
     const setShowAdvertInfo = React.useCallback(
         () => {
-            if (p2p_advert_info) {
-                const { eligibility_status, is_active, is_buy, is_eligible, is_visible } = p2p_advert_info || {};
+            const { is_active, is_buy, is_visible } = p2p_advert_info || {};
+            if (isSubscribed && p2p_advert_info) {
                 const advert_type = is_buy ? 1 : 0;
 
-                if (is_active && is_visible && is_eligible) {
+                if (is_active && is_visible) {
                     advertiser_page_store.setActiveIndex(advert_type);
                     advertiser_page_store.handleTabItemClick(advert_type);
                     buy_sell_store.setSelectedAdState(p2p_advert_info);
                     showModal({ key: 'BuySellModal' });
                 } else {
-                    showErrorModal(eligibility_status);
+                    showErrorModal();
```

Comment:
we still need to pass `eligibility_status` here since we are showing different messages if the ad is unavailable

Reply:
ah i'll place it back 👍

Reply:
should I also add back the is_eligible check in the if statement?